hq/app/logic/ReportDisplay.scala

```diff
@@ -0,0 +1,55 @@
+package logic
+
+import model._
+import org.joda.time.{DateTime, DateTimeZone}
+
+
+object ReportDisplay {
+
+
+  private[logic] def lastActivityDate(cred: IAMCredential): Option[DateTime] = {
+    val allDates =
+      cred.passwordLastUsed.toSeq ++ cred.accessKey1LastUsedDate.toSeq  ++ cred.accessKey2LastUsedDate.toSeq
+    allDates.sortWith(_.isAfter(_)).collectFirst { case date if date.isBefore(DateTime.now(DateTimeZone.UTC)) => date }
+  }
+
+  private def isKey1Disabled(cred: IAMCredential): Boolean = (!cred.accessKey1Active) && cred.accessKey1LastUsedDate.toSeq.nonEmpty
+  private def isKey2Disabled(cred: IAMCredential): Boolean = (!cred.accessKey2Active) && cred.accessKey2LastUsedDate.toSeq.nonEmpty
+
+  private[logic] def key1Status(cred: IAMCredential): KeyStatus = if (cred.accessKey1Active) AccessKeyEnabled else if (isKey1Disabled(cred)) AccessKeyDisabled else NoKey
+  private[logic] def key2Status(cred: IAMCredential): KeyStatus = if (cred.accessKey2Active) AccessKeyEnabled else if (isKey2Disabled(cred)) AccessKeyDisabled else NoKey
+
+  private[logic] def machineReportStatus(cred: IAMCredential): ReportStatus = {
+
+    if (!Seq(key1Status(cred), key2Status(cred)).contains(AccessKeyEnabled)) Amber else Green
+  }
+
+  private[logic] def humanReportStatus(cred: IAMCredential): ReportStatus =
```

Comment:
We've got a heady mix of if statements spread over heaps of lines and some squeezed onto one. For my money, the options in Scala are:

Simple if statement:

    if (something) a else b

Medium complexity if statement:

    if (something a bit more complex) slight more complex a
    else slightly more complex b

Complex if statements:

    if (something) {
      // some logic in here
    } else {
      // logic
    }

Here we have similar if statements with very different displays, and this one with the values on new lines and no braces is a bit unusual.

This is a minor point, but it'd be good to settle on something here.